src/test/components/TrackMemory.test.js

```diff
@@ -66,33 +67,36 @@ describe('TrackMemory', function() {
       .spyOn(HTMLElement.prototype, 'getBoundingClientRect')
       .mockImplementation(() => getBoundingBox(GRAPH_WIDTH, GRAPH_HEIGHT));
 
-    const view = mount(
+    const renderResult = render(
       <Provider store={store}>
         <TrackMemory counterIndex={0} />
       </Provider>
     );
+    const { container } = renderResult;
 
     // WithSize uses requestAnimationFrame
     flushRafCalls();
-    view.update();
 
-    const canvas = view.find('.timelineTrackMemoryCanvas').first();
+    const canvas = ensureExists(
+      container.querySelector('.timelineTrackMemoryCanvas')
```

Comment:
Optional: It would be nice to have friendly error messages for the `ensureExists` calls when running query selectors, especially since they had friendly test failures before, e.g. "Could not select the canvas, did the selector change?". Mainly it'll help future authors when they are refactoring things hone in on the errors. I ran into lots of annoying failures like this when refactoring other DevTools code in the past. At the very least, the `ensureExists` will error out very quickly, even without a message.